vicinity/vicinity.py

```diff
@@ -50,6 +52,17 @@ def __init__(
         self.items: list[str] = list(items)
         self.backend: AbstractBackend = backend
         self.metadata = metadata or {}
+        self.vector_store = vector_store
+
+    def get_vector_by_index(self, index: int | Iterable[int]) -> npt.NDArray:
```

Comment:
What if the index deosn't exist? Should maybe give an informative error and/or be included in our tests?